storm-core/src/jvm/org/apache/storm/trident/operation/builtin/Debug.java

```diff
@@ -40,7 +42,7 @@ public Debug(String name) {
 
     @Override
     public boolean isKeep(TridentTuple tuple) {
```

Comment:
I would like to keep these changes as it is useful to see the timestamp in Debug output. I do not think we need a separate PR for this minor change.